requirements/pip.txt

```diff
@@ -10,5 +10,5 @@ wheel==0.37.1
 # The following packages are considered to be unsafe in a requirements file:
 pip==22.0.2
     # via -r requirements/pip.in
-setuptools==60.5.0
+setuptools==60.7.1
```

Comment:
Please include common constraints in `pip.in` file to observe constraint of `setuptools` from common_constraints.